web/turkey/survey/static/survey/js/auditors/visibility_changes.js

```diff
@@ -0,0 +1,17 @@
+// helper script file for focus_change auditors
```

Comment:
So this is pulled out because it's shared among multiple auditors? Is there an easy way to do mixins with javascript?